### DISCUSSION TITLE
TypeError: 'NoneType' object is not iterable, all plots damage

### DISCUSSION THREAD

**Discussion by glorystarme** [Chia-Network/chia-blockchain#5053] [Support]
Today I routinely checked my plots, chia plots check, and found that all plots were damaged, prompting "'NoneType' object is not iterable", and I couldn't add a new plots folder, and the error prompt was "TypeError: argument of type 'NoneType' is not iterable"
To solve this problem, I restarted the computer and reinstalled chia, but it did not solve the problem. Please help me.
System: windows 10 19041.985 64bit
[![gy2OEQ.png](https://z3.ax1x.com/2021/05/15/gy2OEQ.png)](https://imgtu.com/i/gy2OEQ)
[![gy2XNj.png](https://z3.ax1x.com/2021/05/15/gy2XNj.png)](https://imgtu.com/i/gy2XNj)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
